conf/modules/rank_variants.config

```diff
@@ -45,20 +51,4 @@ process {
         ext.when = false
     }
 
-    withName: '.*:RANK_VARIANTS_SNV:TABIX_BGZIP' {
-        ext.prefix = { "${meta.id}_snv_annotated_ranked" }
-        publishDir = [
-            path: { "${params.outdir}/snvs/multi_sample/${meta.id}" },
-            mode: params.publish_dir_mode,
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
-        ]
-    }
-
-    withName: '.*:RANK_VARIANTS_SNV:TABIX_TABIX' {
-        publishDir = [
-            path: { "${params.outdir}/snvs/multi_sample/${meta.id}" },
-            mode: params.publish_dir_mode,
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
-        ]
-    }
 }
```

Comment:
How come you no longer need this part? Is it to avoid publishing them?

Reply:
Yes, now all files will be published from BCFTOOLS_CONCAT regardless if they're just `calls`, `calls + annotation`, or `calls + annotation + ranked`. 

Before `calls + annotation + ranked` was output from RANK_VARIANTS_SNV:TABIX_BGZIP and RANK_VARIANTS_SNV:TABIX_TABIX, and `calls`, `calls + annotation` from BCFTOOLS_CONCAT.